Makefile

```diff
@@ -448,6 +448,10 @@ ifneq ($(OS), WINNT)
 endif
 ifeq ($(OS), Linux)
 	-$(JULIAHOME)/contrib/fixup-libstdc++.sh $(DESTDIR)$(private_libdir)
+	# We need to bundle ca certs on linux now that we're using libgit2 with ssl
+ifneq ($(shell cat $(shell openssl version -d | cut -d '"' -f 2)/cert.pem),)
+	-cp $(shell openssl version -d | cut -d '"' -f 2)/cert.pem $(DESTDIR)$(datarootdir)/julia/
+endif
```

Comment:
On CentOS 5 where we build the linux tarball binaries, the cert.pem file comes from the openssl package http://mirror.centos.org/centos/5/os/x86_64/CentOS/openssl-0.9.8e-27.el5_10.4.x86_64.rpm